packages/trpc/server/routers/viewer/teams/get.handler.ts

```diff
@@ -15,7 +15,10 @@ type GetOptions = {
 };
 
 export const getHandler = async ({ ctx, input }: GetOptions) => {
-  const team = await getTeamWithMembers({ id: input.teamId, userId: ctx.user.id });
+  const team = await getTeamWithMembers({
+    id: input.teamId,
+    userId: ctx.user.organization?.isOrgAdmin ? undefined : ctx.user.id,
```

Comment:
Thanks for the heads-up @zomars, grabbing org context and admin privileges from `ctx.user`